cmd/batch_test.go

```diff
@@ -23,12 +23,15 @@ type MockBatchPipeline struct {
 	MockWriteTemplate func(
 		writer io.Writer,
 		template string,
-		pregap bool,
+		pregap int,
+		postgap int,
```

Comment:
Ok last change I can see, update the pregap and postgap var names to beforeNewlines and afterNewlines to match the original implementation.